<?xml version="1.0" encoding="utf-8"?>

<launch>
    <node name="dnnCV" pkg="perception" type="dnnCV.py" output="screen" />
    <!-- <include file="$(find darknet_ros)/launch/anafi_real.launch"></include> -->
</launch>
